src/pages/timer.js

```diff
@@ -0,0 +1,24 @@
+import { getTimerFromStorage } from '../lib/storage.js';
+
+import { initResultPage } from './resultPage.js';
+export const createTimePressure = () => {
+  const timer = setInterval(() => {
```

Comment:
It's currently a bit annoying that when one goes to the next question the time is not visible, and then it jumps in after 1 second. This happens because `setInterval` only executes after the defined amount of time for the first time. Best is to execute the function straight away once, and then use `setInterval` to repeatedly trigger it again. Can you adjust that? I think it will make the quiz application feel more stable.

To do this you need to put the arrow-function which is now defined in the `setInterval` in a separate variable, then first call it, and then pass it into `setInterval`. Calling `createTimePressure` also has to move a bit down in the questionPage code, as the progress element first needs to be created and added to the DOM.

Reply:
hello Arco , i did exactly what you told me to do and tried many things also but still the time visible and jumps but i notice that dont need one secont to jump again less than one second and show up , so i think to solve this issue we need to call setinterval method in welcome page not in initialpage function , so in this case what is your suggestion ??